pkg/operator/ceph/cluster/controller_test.go

```diff
@@ -130,3 +134,65 @@ func TestReconcileDeleteCephCluster(t *testing.T) {
 		assert.True(t, kerrors.IsNotFound(err))
 	})
 }
+
+func TestRemoveFinalizers(t *testing.T) {
+	reconcileCephCluster := &ReconcileCephCluster{
+		opManagerContext: context.TODO(),
+	}
+	s := scheme.Scheme
+
+	t.Run("delete finalizer for CephCluster resource", func(t *testing.T) {
```

Comment:
follow similar to this approach when using t.Run()

https://github.com/rook/rook/blob/08cb6785a4e4c2dc67c4443a37def17aa3ba5617/pkg/apis/ceph.rook.io/v1/pool_test.go#L68-L87

Reply:
Changed, it looks more better.